packages/generator/src/reserved-words.ts

```diff
@@ -71,4 +61,15 @@ export const reservedServiceKeywords: string[] = [
   'Link',
   'Time',
   'TimeField'
-];
+] as const;
```

Comment:
just out of curiosity what is the `as const` doing? The type is redundant, that is why you removed it?

Reply:
1. const makes this readonly - that avoids us accidentally adding anything to it (because this is an array and if you pass it is passed by reference). I am not 100% sure that this is the best way to achieve this, btw.
(so the type here becomes `readonly string[]`)
2. The type was `Set` before and it was used inconsistently. I don't see a good reason for it to be a set and made it an array instead (and then I left out the type annotations ;) )